src/BoundaryConditions/field_boundary_conditions.jl

```diff
@@ -193,9 +193,12 @@ boundary conditions for prognostic model field boundary conditions.
 function regularize_field_boundary_conditions(bcs::FieldBoundaryConditions,
                                               grid::AbstractGrid,
                                               field_name::Symbol,
+                                              open_boundaries=(; west = false, east = false, south = false, north = false, bottom = false, top = false),
```

Comment:
What's this?

Reply:
This was a (non working) attempt to automatically add other open conditions when one is specified (i.e. detect that the user has put an open boundary and add appropriate boundaries to all other tracers). But I think this is a challenge for later (and doesn't have a clear answer because it would be a subjective choice what the other ones should be filled with).